SRS.md

```diff
@@ -1,28 +1,98 @@
+## 2. Employees  
+  ### 2.1. Display list of Employees  
+  This mode is intended for viewing and editing the employees list.  
+  
+  __Main scenario:__
+  1. User selects item "Employees";
+  2. Application dispalys list of employees.  
+      
+  ![image](https://user-images.githubusercontent.com/83345134/127285130-2928a7fa-dd6e-4b84-a077-ff80b723aa25.png "Employees list")  
+    Pic 2.1 View the employees list.
+    
+  __The list displays following employees' information:__  
+  1. Department - place, where employee is working;  
+  2. Name of the employee;  
+  3. Date of birth of the employee;  
+  4. Employee's salary.  
+    
+  __Filtering by date:__  
+  1. In the employees list view mode, the user sets a date filter and presses the refresh list button (to the right of the date entry field);  
+  2. The application will show the employees only for a certain period of time.  
```

Comment:
1. How to reset a search result?
2. What if we need to find employees who were born after a certain date?
3. What if we need to find employees who were born before a certain date?

Reply:
fixed